test/performance/dataplane-probe/main.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/mako/helpers/go/quickstore"
+	qpb "github.com/google/mako/helpers/proto/quickstore/quickstore_go_proto"
+	vegeta "github.com/tsenart/vegeta/lib"
+	"knative.dev/pkg/signals"
+
+	"knative.dev/serving/test/performance/mako"
+)
+
+var (
+	target = flag.String("target", "", "The target to attack.")
+)
+
+func main() {
+	flag.Parse()
+
+	// We want this for properly handling Kubernetes container lifecycle events.
+	ctx := signals.NewContext()
+
+	// The mako sidecar address
+	addr := "localhost:9813"
+
+	// We cron every 10 minutes, so give ourselves 6 minutes to complete.
+	ctx, cancel := context.WithTimeout(ctx, 6*time.Minute)
+	defer cancel()
+
+	// Use the benchmark key created
+	q, close, err := quickstore.NewAtAddress(ctx, &qpb.QuickstoreInput{
+		BenchmarkKey: proto.String("5142965274017792"),
+		Tags:         []string{"master"},
```

Comment:
cc @srinivashegde86 

We should consider changing this to include: https://github.com/knative/pkg/blob/8a10634b4fa4df6dc450bfd883e85a33e76f2471/changeset/commit.go#L39